src/libtriton/includes/pythonBindings.hpp

```diff
@@ -14,6 +14,8 @@
   #include <python2.7/Python.h>
   #include <python2.7/longintrepr.h>
 #elif _WIN32
+  #include <cmath>
+  #define _hypot hypot
```

Comment:
mingw is giving errors in python.h if I leave these out. I figured out that I need both of these lines to avoid the error (otherwise it complains about either missing hypot or _hypot).